scripts/write_deployinst_input.py

```diff
@@ -217,63 +217,39 @@ def reactor_render(list, output_file, is_cyborg=False):
     pwr_template = read_template(template_path.replace('[reactor]', 'pwr'))
     mox_reactor_template = read_template(template_path.replace('[reactor]', 'mox'))
     candu_template = read_template(template_path.replace('[reactor]', 'candu'))
+
+    spec_dict = {}
+    # spec_dict[ 'reactor_type'] = [template, assem_size, n_assme_core(for model), n_assem_batch(for model)]
+    spec_dict['BWR'] = [pwr_template, 180, 764 / 1000, 764 / 3000]
```

Comment:
integer/integer division results in an integer in python 2, but a float in python 3. If you want a float (I think you do), you must use 764./1000. instead.

Reply:
(If you used unit tests and travis to test this code, you would have been informed of this automatically. Perhaps now is a good time to add unit tests!!!)

Reply:
I think it is required to have integers for `n_assem_core` and `n_assem_batch`

Reply:
then you should be careful because in python 3 they won't be integers, they'll be floats.